<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_PG_DamageParameter_Add</class>
 <widget class="QDialog" name="Frm_PG_DamageParameter_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加/编辑毁伤参数</string>
  </property>
  <widget class="QGroupBox" name="gb_scene_info">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>491</width>
     <height>109</height>
    </rect>
   </property>
   <property name="title">
    <string>场景信息</string>
   </property>
   <widget class="QLabel" name="lbl_scene_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>场景编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_scene_code">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>35</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>（必填）</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_required_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_scene">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择场景</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scene_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>场景ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_scene_id">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>71</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_basic_params">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>124</y>
     <width>491</width>
     <height>108</height>
    </rect>
   </property>
   <property name="title">
    <string>基本参数</string>
   </property>
   <widget class="QLabel" name="lbl_carrier">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>投放平台：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_carrier">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>30</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_guidance_mode">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>32</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>制导方式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_guidance_mode">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>30</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_warhead_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>战斗部类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_warhead_type">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>68</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>爆破战斗部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>聚能穿甲战斗部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>破片战斗部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>钻地侵彻战斗部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>子母弹战斗部(爆破子弹)</string>
     </property>
    </item>
        <item>
     <property name="text">
      <string>子母弹战斗部(聚能穿甲子弹)</string>
     </property>
    </item>
        <item>
     <property name="text">
      <string>子母弹战斗部(破片子弹)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_required_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_charge_amount">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>装药量(kg)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_charge_amount">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>70</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_drop_params">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>238</y>
     <width>491</width>
     <height>109</height>
    </rect>
   </property>
   <property name="title">
    <string>投放参数</string>
   </property>
   <widget class="QLabel" name="lbl_drop_height">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>投弹高度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_drop_height">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>34</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_drop_speed">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>34</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>投弹速度(m/s)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_drop_speed">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>34</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_drop_mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>69</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>投弹方式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_drop_mode">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>69</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_flight_range">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>69</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>射程(km)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_flight_range">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>69</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_environment">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>353</y>
     <width>491</width>
     <height>109</height>
    </rect>
   </property>
   <property name="title">
    <string>环境参数</string>
   </property>
   <widget class="QLabel" name="lbl_electro_interference">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电磁干扰等级：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_electro_interference">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>低</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_weather_conditions">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>32</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>天气状况：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_weather_conditions">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>晴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>多云</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>阴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>雨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>雪</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_wind_speed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>环境风速(m/s)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_wind_speed">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>70</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>470</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
